src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -7,8 +7,10 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
 {
     internal static class TimeSpanExtensions
     {
+        private const int MaxAttempts = 63;
```

Comment:
The reason we use 63 max attempts is because 1 << 63 is the first value that overflows. It has nothing to do with the max number of attempts or retries. Attempts can keep growing beyond 63 but `MaxAttempts` is just used to limit the max backoff period for a specific attempt.